UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Startup.cs

```diff
@@ -304,5 +309,32 @@ void ConfigAdditionalValuesProvider(IDictionary<string, object> additionalValues
             app.UseCorrelationIdMiddleware()
                .UseErrorLogging(loggerFactory);
         }
+
+        private IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(ILogger logger, string requestType, EventIds eventId)
+        {
+
+            var retryCount = Convert.ToInt32(configuration["RetryConfiguration:RetryCount"]);
+            var sleepDuration = Convert.ToDouble(configuration["RetryConfiguration:SleepDuration"]);
+            return Policy
+                .HandleResult<HttpResponseMessage>(r => r.StatusCode == HttpStatusCode.ServiceUnavailable || r.StatusCode == HttpStatusCode.TooManyRequests)
```

Comment:
it looks like me you have the or defined twice.  You need either
```cs
.HandleResult<HttpResponseMessage>(r => r.StatusCode == HttpStatusCode.ServiceUnavailable || r.StatusCode == HttpStatusCode.TooManyRequests)
```

or
```cs
.HandleResult<HttpResponseMessage>(r => r.StatusCode == HttpStatusCode.ServiceUnavailable)
.OrResult(r => || r.StatusCode == HttpStatusCode.TooManyRequests)
````

Reply:
did changes accordingly